.claude/settings.local.json

```diff
@@ -0,0 +1,12 @@
+{
+  "permissions": {
+    "allow": [
+      "Bash(trunk check:*)",
+      "Bash(git checkout:*)",
+      "Bash(find:*)",
+      "Bash(git mv:*)",
+      "Bash(git add:*)"
+    ],
+    "deny": []
+  }
+}
```

Comment:
I'm thinking that we don't want to check these in... Each developer should have their own discrepancy around which commands they feel comfortable being always allowed. What do you think? cc @oycyc